alica_engine/src/engine/BasicBehaviour.cpp

```diff
@@ -112,6 +113,7 @@ void BasicBehaviour::initJob()
 
     // Todo: Optimization: don't call initialiseParameters if not in context
     try {
+        auto trace = IAlicaTrace("Init", _trace->context());
```

Comment:
there is no need to create traces for init/terminate. The trace should be created in start() & destroyed in stop(). For init, terminate & the first run() call set corresponding boolean tags on the trace i.e. `_trace->setTag("init", "true")`.